### ISSUE TITLE
Pilot user research for IncLim

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#12340]
## Description
Goal: Run pilots for the income limits research.

## Acceptance Criteria
- [x] Recruit pilot participant(s)
- [x] Schedule pilot session(s)
- [x] Test Zoom login from Perigean and change display name of Perigean Zoom account to "Cindy (Moderator)"
- [x] Send instructions to observers of the pilot
- [x] Run pilot session(s)
- [x] Debrief pilot session(s)
- [x] Send thank you notes to pilot participants
- [x] Revise conversation guide from pilot session(s)
- [ ] If needed, revise prototype from pilot session(s)
- [x] Decide if will address issues raised last week:
  - [x] Get correct income numbers for past year flow--NO
  - [x] Change income limits to salary ranges--YES

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by cindymerrill** [comment]
BLOCKED by Content

**Comment by cindymerrill** [comment]
Scheduled 2 pilot participants for Mon and Tue.

**Comment by cindymerrill** [comment]
Shared observer instructions with Wes and Jordan

**Comment by cindymerrill** [comment]
Postponed today's pilot test to Wed, per participant's request

**Comment by cindymerrill** [comment]
Ran first pilot test today. Made some changes to conversation guide and prototype so ready for the second pilot test tomorrow.

**Comment by cindymerrill** [comment]
Ran second pilot. Discussed a few changes to the prototype and convo guide, which can happen tomorrow.

**Comment by cindymerrill** [comment]
@thejordanwood Made some changes to the prototype from the pilots. There are two open issues left, which Jordan can work on when she gets back from PTO while I'm out on PTO:

1. Higher placement of the "Review the information you entered" link on the review screen in both present and past flows. I recommend putting the link just below the breadcrumb (because it's navigation), but Jordan thought maybe just above the "Select your 2022 household income range" header.
2. (new!) Fix typos that I noticed in Past flow results: 
    - Some of the salary ranges overlap. Need to change two of the endpoints so they're all unique.
    - (super minor, but if you're editing the design anyway) There's an extra space in the "how to estimate your income for this tool" accordion after "for a spouse or child".

**Comment by cindymerrill** [comment]
@jilladams @wesrowe I made convo guide changes from the pilot, and Jordan made some changes to the prototype but isn't totally finished. I've detailed the changes still needed in my previous comment (see above), and Jordan won't be able to work on them until she gets back from PTO on Mar 27 (when I'm on PTO). Not sure if you want to keep this ticket open or save the prototype design feedback in a new ticket. I just don't want it to get lost, but I also don't want to tie up the sprint because this ticket is a sprint goal for Research. Should I copy the prototype feedback into Jordan's [prototype ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12343)?

**Comment by wesrowe** [comment]
@cindymerrill, Jordan has [her own ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12343) for adapting the prototype based on pilot learnings. I suggest adding comments there.

**Comment by cindymerrill** [comment]
@wesrowe OK, I added my prototype comments to Jordan's design ticket, and so I'll close this ticket. When I get back from PTO, I'll review the prototype, which I'll add to my [Research Prep Tasks ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12256).
